cmd/erasure-common.go

```diff
@@ -55,9 +54,13 @@ func (er erasureObjects) getLoadBalancedNDisks(ndisks int) (newDisks []StorageAP
 // getLoadBalancedDisks - fetches load balanced (sufficiently randomized) disk slice.
 func (er erasureObjects) getLoadBalancedDisks() (newDisks []StorageAPI) {
 	disks := er.getDisks()
+
 	// Based on the random shuffling return back randomized disks.
 	for _, i := range hashOrder(UTCNow().String(), len(disks)) {
-		newDisks = append(newDisks, disks[i-1])
+		// Do not consume disks which are being healed.
```

Comment:
I checked all the callers and this seems safe for these.

But please add to the function documentation that it excludes healing disks.

Reply:
Sure @klauspost